/**
 * Created by qfeng on 2018/12/29
 */

syntax = "proto3";

option cc_enable_arenas = true;
option java_outer_classname = "AUCImprFeatsPb";
option java_multiple_files = true;
option java_package = "algocomp";

import "featpb/feature_bunch.proto";

package algocomp;

message AUCImprFeats {
    repeated float labels = 1;
    repeated float weights = 2;
    repeated float scores = 3;
    FeatureBunch user_and_ctx = 4;
    repeated FeatureBunch ads = 5;
}